src/NonStrippingCreditCard.php

```diff
@@ -28,4 +35,100 @@ public function setNumber($value)
          */
         return $this->setParameter('number', $value);
     }
+
+    /**
+     * Sets the payment instrument name.
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setPaymentInstrumentName($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('paymentInstrumentName', $value);
+    }
+
+    /**
+     * Gets the payment instrument name.
+     *
+     * @return string|null
+     */
+    public function getPaymentInstrumentName()
+    {
+        return $this->getParameter('paymentInstrumentName');
+    }
+
+    /**
+     * Sets the payment network.
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setPaymentNetwork($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('paymentNetwork', $value);
+    }
+
+    /**
+     * Gets the payment network.
+     *
+     * @return string|null
+     */
+    public function getPaymentNetwork()
+    {
+        return $this->getParameter('paymentNetwork');
+    }
+
+    /**
+     * Sets the transaction identifier.
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setTransactionIdentifier($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('transactionIdentifier', $value);
+    }
+
+    /**
+     * Gets the transaction identifier.
+     * 
+     * @return string|null
+     */
+    public function getTransactionIdentifier()
+    {
+        return $this->getParameter('transactionIdentifier');
+    }
+
+    /**
+     * Sets the country, zip code, expiration date and account holder name.
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setPaymentData($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('paymentData', $value);
+    }
+
+    /**
+     * Gets the country, zip code, expiration date and account holder name.
```

Comment:
where'd you find that this was what this field contains? the examples i've seen, including the one in the test below, look like a bunch of encrypted stuff.

do we actually need this field in Omnipay? if it is address information, we already have that elsewhere on the card object. if it's not address info, not sure it's super useful.

Reply:
It is an address info but this field is only way we are going to get it

Reply:
if the address info really doesn't come back from vindicia other than in this field, we should parse this field in this library and set the information on the address fields on the card in the AbstractRequest::buildPaymentMethod function. so that would be part of the next PR, but this field still wouldn't be necessary on this card object.